### DISCUSSION TITLE
Latest version of preview 1.15 still not available in winget after 2 weeks

### DISCUSSION THREAD

**Discussion by jessey-git** [microsoft/terminal#13542] [Q&A]
### Windows Terminal version

1.14.1432.0

### Windows build number

10.0.19044.0

### Steps to reproduce

- Have version 1.14.x of Microsoft.WindowsTerminal.Preview installed by winget
- Try to upgrade to the latest preview release using winget
- Be sad when it doesn't upgrade even though 1.15 has been out for 12 days

### Expected Behavior

`winget` gets the packages at the exact same time as the Windows Store.

### Actual Behavior

```
> winget list Terminal
Name                     Id                                Version     Available Source
---------------------------------------------------------------------------------------
Windows Terminal Preview Microsoft.WindowsTerminal.Preview 1.14.1432.0           winget

> winget upgrade Microsoft.WindowsTerminal.Preview
No applicable update found.
```

Did some machinery fail to publish to winget?  The Windows Store is showing the "Update" button, but I really don't want to give them any MAUs.

**Comment by zadjii-msft** [comment]
I'll make sure we go update that. Thanks for the heads up! In the meantime, you can always download the package directly off our Releases page.

**Reply by jessey-git** [reply]
Guessing that something has come up? Seems like winget still isn't listing the version

**Reply by zadjii-msft** [reply]
Yea, we've had a lot of people out unexpectedly. We're trying to get a hotfix out the door for a couple issues, so it seems like a bad idea to waste cycles on updating the current winget package to one we plan on also replacing Soon<sup>tm</sup>. Sorry about that!

**Comment by jessey-git** [comment]
Looks like it was updated yesterday!
